README.md

```diff
@@ -0,0 +1,19 @@
+[![Build Status](https://travis-ci.org/wo-ist-markt/wo-ist-markt-berlin-update-reminder.svg?branch=master)](https://travis-ci.org/wo-ist-markt/wo-ist-markt-berlin-update-reminder)
+
+# Wo ist Markt? Berlin update reminder
+
+A maintenance script for the [Wo ist Markt? project](https://github.com/wo-ist-markt/wo-ist-markt.github.io)
+to automatically check if the market data for Berlin has changed.
+
+
+## Project description
+
+The script compares two time stamps. One can be found on the website for
+the Berlin market data on the data portal. The counterpart can be found at the
+bottom of the JSON file which is rendered at the website.
```

Comment:
which website is the last sentence referring to? Also maybe it makes sense to add links to both sources?

Reply:
I was not sure if it is smart to duplicate the links as the script already hardcodes them.

Reply:
I think it would be better then to tell people to read the source.

Reply:
@k-nut Fixed in 280ccd1c62130f7d54b59995ad47a7ef3e28e824.